plugins/catalog-react/api-report.md

```diff
@@ -279,6 +295,21 @@ export type EntityPeekAheadPopoverProps = PropsWithChildren<{
   delayTime?: number;
 }>;
 
+// @public
+export interface EntityPresentationApi {
+  forEntity(
+    entityOrRef: Entity | string,
+    context?: {
+      variant?: string;
```

Comment:
I think we should do something more explicit with this variant tbh, or simply skip it for now. As a plugin author I don't really know what values are valid here. I'm feeling the minimum would be something like `'icon' | string`